app/lib/finance/api/response.rb

```diff
@@ -1,6 +0,0 @@
-module Finance
-  module API
-    module Collection
-    end
-  end
-end
```

Comment:
what was this?

Reply:
I don't know, check this comment: https://github.com/3scale/porta/pull/3339#discussion_r1242428237

I think it was a mistake of leftover.